File: ModelLibrary/Models/Candidate.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace ModelLibrary.Models
{
    public class Candidate
    {
        [Key]
        int candidateId { get; set; }
        string firstName { get; set; }
        string lastName { get; set; }
        string picture { get; set; }
        string biography { get; set; }

        string organizationName { get; set; }
        Organization organization { get; set; }

        public List<CandiateRace> CandidateRaces { get; } = new List<CandiateRace>();
    }
}

File: Spa/ClientApp/src/app/models/candidate.ts
export class Candidate {
  candidateId : number;
  firstName   : string;
  lastName    : string;
  picture     : string;
  organization: string;
  selected: boolean;

  constructor(
    candidateId : number,
    firstName   : string,
    lastName    : string,
    picture     : string,
    organization: string,
    selected    : boolean
  ) {
    this.candidateId  = candidateId;
    this.firstName    = firstName;
    this.lastName     = lastName;
    this.picture      = picture;
    this.organization = organization;
    this.selected     = false;
  }
}
